lib/src/main/java/uk/gov/di/ipv/cri/passport/library/config/ConfigurationVariable.java

```diff
@@ -1,31 +1,29 @@
-    BACKEND_SESSION_TTL("/%s/credentialIssuers/ukPassport/self/backendSessionTtl"),
-    DCS_POST_URL_PARAM("/%s/credentialIssuers/ukPassport/dcs/postUrl"),
-    DCS_ENCRYPTION_CERT("/%s/credentialIssuers/ukPassport/dcs/encryptionCertForPassportToEncrypt"),
-    DCS_SIGNING_CERT("/%s/credentialIssuers/ukPassport/dcs/signingCertForPassportToVerify"),
-    DCS_TLS_INTERMEDIATE_CERT("/%s/credentialIssuers/ukPassport/dcs/tlsIntermediateCertificate"),
-    DCS_TLS_ROOT_CERT("/%s/credentialIssuers/ukPassport/dcs/tlsRootCertificate"),
-    JAR_ENCRYPTION_KEY_ID("/%s/credentialIssuers/ukPassport/self/jarKmsEncryptionKeyId"),
-    JAR_KMS_PUBLIC_KEY("/%s/credentialIssuers/ukPassport/self/jarKmsEncryptionPublicKey"),
-    MAX_JWT_TTL("/%s/credentialIssuers/ukPassport/self/maxJwtTtl"),
-    MAXIMUM_ATTEMPT_COUNT("/%s/credentialIssuers/ukPassport/self/maximumAttemptCount"),
-    PASSPORT_CRI_CLIENT_AUDIENCE("/%s/credentialIssuers/ukPassport/self/audienceForClients"),
-    PASSPORT_CRI_CLIENT_AUTH_MAX_TTL("/%s/credentialIssuers/ukPassport/self/maxJwtTtl"),
-    PASSPORT_CRI_CLIENT_VC_MAX_TTL("/%s/credentialIssuers/ukPassport/self/MaxVCJwtTtlMapping"),
-    PASSPORT_CRI_CLIENT_TTL_UNIT("/%s/credentialIssuers/ukPassport/self/JwtTtlUnit"),
+    AUTH_CODE_EXPIRY_CODE_SECONDS("/%s/AuthCodeExpirySeconds"),
+    SESSION_TTL("/%s/SessionTtl"),
+    DCS_POST_URL_PARAM("/%s/DCS/PostUrl"),
+    HTTPCLIENT_TLS_CERT("/%s/DCS/HttpClient/TLSCert"),
+    HTTPCLIENT_TLS_KEY("/%s/DCS/HttpClient/TLSKey"),
+    HTTPCLIENT_TLS_INTERMEDIATE_CERT("/%s/DCS/HttpClient/TLSIntermediateCertificate"),
+    HTTPCLIENT_TLS_ROOT_CERT("/%s/DCS/HttpClient/TLSRootCertificate"),
+    PASSPORT_CRI_SIGNING_CERT(
+            "/%s/DCS/JWS/SigningCertForDcsToVerify"), // JWS SHA-1 Certificate Thumbprint (Header)
+    PASSPORT_CRI_SIGNING_KEY("/%s/DCS/JWS/SigningKeyForPassportToSign"), // JWS RSA Signing Key
+    DCS_ENCRYPTION_CERT("/%s/DCS/JWE/EncryptionCertForPassportToEncrypt"), // JWE (Public Key)
+    DCS_SIGNING_CERT("/%s/DCS/JWE/SigningCertForPassportToVerify"), // DCS JWS (Reply Signature)
     PASSPORT_CRI_ENCRYPTION_KEY(
-            "/%s/credentialIssuers/ukPassport/self/encryptionKeyForPassportToDecrypt"),
-    PASSPORT_CRI_SIGNING_CERT("/%s/credentialIssuers/ukPassport/self/signingCertForDcsToVerify"),
-    PASSPORT_CRI_SIGNING_KEY("/%s/credentialIssuers/ukPassport/self/signingKeyForPassportToSign"),
-    PASSPORT_CRI_TLS_CERT("/%s/credentialIssuers/ukPassport/self/tlsCert"),
-    PASSPORT_CRI_TLS_KEY("/%s/credentialIssuers/ukPassport/self/tlsKey"),
-    VERIFIABLE_CREDENTIAL_ISSUER(
-            "/%s/credentialIssuers/ukPassport/self/verifiableCredentialIssuer"),
-    VERIFIABLE_CREDENTIAL_SIGNING_KEY_ID(
-            "/%s/credentialIssuers/ukPassport/self/verifiableCredentialKmsSigningKeyId");
+            "/%s/DCS/JWE/EncryptionKeyForPassportToDecrypt"), // DCS JWE (Private Key Reply Decrypt)
+    JAR_ENCRYPTION_KEY_ID("/%s/AuthRequestKmsEncryptionKeyId"),
```

Comment:
Can we check is there a convention should params be named like this like classes or should they start small and be camel cased

Reply:
Also should align with any common naming to make the swap easier where we can

Reply:
There is heavy inconsistency in template variable between all the CRI's and in the common-framework.

The template variables are named consistently with this inconsistency if you will, so when swapped they should all be read from the same location.

In common-lambda template, variables are majority Title Case, just after the stack name.
jwtAuthentication related paths are lower case with variables in camel case aligned with other all other CRI's and common-lambda.

The passport specific variables are Title Case, just after the stack name most will be replaced with common-lambdas and match up with them common lambdas locations.

DCS params are Title Case but sorted out into paths with 3 letter acronyms to help different the different keys and certs from each other.